### PR TITLE
More and more crafting

### PR DESCRIPTION
**Pull request by roboticxlink**
### Adds the following features:
 - **Tap Syncing Remotes to Collars.** Simply tap the collar with a remote in your hand to sync the remote to the frequency and code! 
- **Broc and Xander seeds added to Seed-Vendors.** 3 of each have been added to MegaSeed servitors.
- **Slave Collar Crafting.** Add a signaling assembly, and an igniter onto a pet collar (crafted with cloth) to make a new item/Slave Collar!
- **Craft Muzzle.** Also adds ability to craft muzzles.

### Bonfire Changes:
- **Bonfire by Logs.** Crafting menu recipe has been changed to 5 towercaps.
- **Wood Crafting Bonfire.** Adds bonfires to the wood crafting list, but now costs 15 Wood Planks.
- **Bonfires Now Have Own Structure .dm** Bonfires were a thing in towercaps' plant file. It is now moved to its own file in the proper location.

### CODE REVIEW

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 1]
```diff
@@ -199,6 +199,7 @@ GLOBAL_LIST_INIT(wood_recipes, list ( \
 	new/datum/stack_recipe("ore box", /obj/structure/ore_box, 4, time = 50, one_per_turf = TRUE, on_floor = TRUE),\
 	new/datum/stack_recipe("wooden crate", /obj/structure/closet/crate/wooden, 6, time = 50, one_per_turf = TRUE, on_floor = TRUE),\
 	new/datum/stack_recipe("baseball bat", /obj/item/twohanded/baseball, 5, time = 15),\
+	new/datum/stack_recipe("bonfire", /obj/structure/bonfire, 15, time = 60),\
```
Comment (AndrewL97) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: one_per_turf = TRUE

Just to stop stacking bonfires

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 2]
```diff
@@ -250,6 +251,8 @@ GLOBAL_LIST_INIT(cloth_recipes, list ( \
 	new/datum/stack_recipe("black gloves", /obj/item/clothing/gloves/color/black, 3), \
 	null, \
 	new/datum/stack_recipe("blindfold", /obj/item/clothing/glasses/sunglasses/blindfold, 2), \
+	new/datum/stack_recipe("muzzle", /obj/item/clothing/mask/muzzle, 2), \
```
Comment (UndeadNappist) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: /obj/item/clothing/mask/muzzle/gag
Doesn't have to be anything different, but sets up for a new icon_state if you want a cloth gag.

File: code/game/objects/structures/bonfire.dm [hunk 3]
```diff
@@ -0,0 +1,153 @@
+/////////BONFIRES//////////
```
Comment (UndeadNappist) [inline code/game/objects/structures/bonfire.dm]: #176 - there aren't any conflicts, are there?

File: code/modules/crafting/recipes.dm [hunk 4]
```diff
@@ -724,5 +724,11 @@
...
 	category = CAT_DRUGS
 
-
-
+/datum/crafting_recipe/slavecollar
+	name = "Slave Collar"
+	result = /obj/item/electropack/shockcollar
+	reqs = list(/obj/item/clothing/neck/petcollar = 1,
+				/obj/item/assembly/signaler = 1,
+				/obj/item/assembly/igniter =1)
```
Comment (UndeadNappist) [inline code/modules/crafting/recipes.dm]: /obj/item/assembly/igniter = 1 instead
igniter/signaler assembly?

Comment (LetterN) [reply code/modules/crafting/recipes.dm]: igniter makes sense

File: code/game/objects/structures/bonfire.dm (hunk 3 above)
Comment (LetterN) [reply code/game/objects/structures/bonfire.dm]: this was just moved into it's folder

**Comment by roboticxlink** [comment]
Is this good now?

**Comment by imw-Oculus** [comment]
Looks fine to me.

**Pull request merged**